### PR TITLE
Ls react issue2

### PR DESCRIPTION
**Pull request by zju2**
Closes React Issue2

### CODE REVIEW

**Comment by zju2** [comment]
[React Issue 2: Add FeedbackStore, and add FeedbackForm](https://trello.com/c/F1totIXx/461-react-issue-2-add-feedbackstore-and-add-feedbackform)

File: app/javascript/image_sharing/test/stores/FeedbackStore.spec.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const assert = require('assert');
+
+describe('test my FeedbackStore', () => {
+  it('Initialization should work correctly', () => {
+    const store = new FeedbackStore();
+    assert.strictEqual(store.name, '');
+    assert.strictEqual(store.comment, '');
+  });
+
+  it('SetName should work correctly', () => {
```
Comment (natalieschauser) [inline app/javascript/image_sharing/test/stores/FeedbackStore.spec.js]: very minor, but these test names should sound like an english sentence. So changing it to be something like:
`it('should set name correctly', () => {` would be better than `it('SetName should work correctly', () => {`.

This comment applies to all the other react tests that you wrote in this PR as well.

File: app/javascript/image_sharing/test/components/Form.spec.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  it('should call setName when name input changes', () => {
+    const page = shallow(<Form stores={stores} />).dive();
+    const nameRow = page.find('input');
+
+    let event = { target: { value: 'Alice' } };
+    nameRow.simulate('change', event);
+    assert(stores.feedbackStore.setName.calledWith('Alice'));
+
+    event = { target: { value: 'Bob' } };
```
Comment (natalieschauser) [inline app/javascript/image_sharing/test/components/Form.spec.js]: What was the reason for testing multiple change events for `nameRow`? I think just testing that it is called with the correct arguments once is good enough.

File: app/javascript/image_sharing/test/components/Form.spec.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+  it('should call setComment when comment input changes', () => {
+    const page = shallow(<Form stores={stores} />).dive();
+    const commentRow = page.find('textarea');
+
+    let event = { target: { value: 'Good!' } };
+    commentRow.simulate('change', event);
+    assert(stores.feedbackStore.setComment.calledWith('Good!'));
+
+    event = { target: { value: 'Bad!' } };
```
Comment (natalieschauser) [inline app/javascript/image_sharing/test/components/Form.spec.js]: Same here :)

File: app/javascript/image_sharing/test/components/App.spec.js [hunk 4]
```diff
@@ -14,10 +15,19 @@ describe('test the App', () => {
     const stores = {
       feedbackStore: new FeedbackStore()
     };
-    const app = shallow(<App stores={stores} />).dive();
+    const app = shallow(<App stores={stores} />);
```
Comment (natalieschauser) [inline app/javascript/image_sharing/test/components/App.spec.js]: You no longer need `stores` as a prop in `App` right? Because you deleted the `inject`.

Comment (zju2) [reply app/javascript/image_sharing/test/components/App.spec.js]: oh my bad, I forget to remove `stores` here and next test. Thanks, Natalie!

File: app/javascript/image_sharing/test/stores/FeedbackStore.spec.js (hunk 1 above)
Comment (zju2) [reply app/javascript/image_sharing/test/stores/FeedbackStore.spec.js]: Will do, thanks

File: app/javascript/image_sharing/test/components/Form.spec.js (hunk 2 above)
Comment (zju2) [reply app/javascript/image_sharing/test/components/Form.spec.js]: Got it, thanks

**Pull request merged**